Documentation/README_MRTKStandardShader.md

```diff
@@ -105,18 +107,55 @@ Performant plane, sphere, and box shape clipping with the ability to specify whi
 
 ![primitive clipping gizmos](../Documentation/Images/MRTKStandardShader/MRTK_PrimitiveClippingGizmos.gif)
 
+### Mesh Outlines
+
+Many mesh outline techniques are done using a [post processing](https://docs.unity3d.com/Manual/PostProcessingOverview.html) technique. Post processing provides great quality outlines, but can be prohibitively expensive on mobile Mixed Reality devices. You can find a scene that demonstrates usage of mesh outlines in the  **OutlineExamples** scene under: [MixedRealityToolkit.Examples/Demos/StandardShader/Scenes/](https://github.com/microsoft/MixedRealityToolkit-Unity/tree/mrtk_release/Assets/MixedRealityToolkit.Examples/Demos/StandardShader/Scenes)
+
+<img src="../Documentation/Images/MRTKStandardShader/MRTK_MeshOutline.jpg" width="900">
+
+[**MeshOutline.cs**](xref:Microsoft.MixedReality.Toolkit.Utilities.MeshOutline) and [**MeshOutlineHierarchy.cs**](xref:Microsoft.MixedReality.Toolkit.Utilities.MeshOutlineHierarchy) can be used render an outline around a mesh renderer. Enabling this component introduces an additional render pass of the object being outlined, but is designed to run performantly on mobile Mixed Reality devices and does not utilize any post processes. Limitations of this effect include it not working well on objects which are non-uniformly scaled and depth sorting issues on overlapping objects.
```

Comment:
Can you actually switch these from [**Blah**] to [ \` blah \`  ]  for styling?

Reply:
Certainly!